torch/csrc/jit/runtime/register_distributed_ops.cpp

```diff
@@ -64,13 +64,29 @@ RegisterOperators reg_rpc_ops({
         },
         aliasAnalysisFromSchema()),
      Operator(
-         "aten::confirmed_by_owner(RRef(t) self) -> bool",
-         [](Stack& stack) {
-           auto rref = pop(stack).toRRef();
-           push(stack, rref->confirmedByOwner());
-           return 0;
-         },
-         aliasAnalysisFromSchema()),
+       "aten::owner(RRef(t) self) -> int",
```

Comment:
Do you need to add these two new APIs to the white list in `https://github.com/pytorch/pytorch/blob/master/test/backward_compatibility/check_backward_compatibility.py`?

Reply:
Added.